### DISCUSSION TITLE
[Feature] Docker setup improvements

### DISCUSSION THREAD

**Discussion by mgogoulos** [mediacms-io/mediacms#613] [Feature Requests]
For the Docker setup these need be done:

- [ ] make sure admin password is generated and not the default one is in place
- [ ] SECRET_KEY is auto-generated too
- [x] Improve documentation, specifically how to use it
- [ ]  improve how environment variables are retrieved

**Comment by mgogoulos** [comment]
@swiftugandan suggest more, and feel free to contribute ;)

**Comment by MikeyYeahYeah** [comment]
I would be more than happy to help with this effort.  Just let me know :-)

**Comment by swiftugandan** [comment]
@MikeyYeahYeah please feel free to pick up any of the tasks, would be nice to have a docker deployment peer to bounce ideas off ;)

**Comment by EricGuic** [comment]
Hi, new here, and not a developper.
But a movie producer who try to self-host a maximum of tools I use on a daily basis.
So I own a Synology Nas, with Docker. I can test the native container install, or via Portainer if you need.
Best regards.

**Comment by mgogoulos** [comment]
Hi Eric, please do any testing you can make and report issues. Also would be great if you can write a blog post or similar documenting your experience!

**Comment by falense** [comment]
Hello, 

depends_on lines cause trouble while trying to follow the basic setup instruction (https://github.com/mediacms-io/mediacms/blob/main/docs/admins_docs.md#3-docker-installation) for ubuntu20.04 with docker: 

```
ERROR: The Compose file './docker-compose.yaml' is invalid because:
services.migrations.depends_on contains an invalid type, it should be an array
```

This might be relevant: https://github.com/moby/moby/issues/30404

```    
depends_on:
     redis:
        condition: service_healthy
      db:
        condition: service_healthy
```
Commenting them out leaves me with a working docker-compose file and "docker-compose up" starts setup.

The installation eventually fails with a lot of these:

```requests.exceptions.ReadTimeout: UnixHTTPConnectionPool(host='localhost', port=None)```

Suggestions?
